ADMVEDU95/ADMVEDU95/Presentation/LogIn/View/LoginViewController.swift

```diff
@@ -78,8 +80,8 @@ extension LoginViewController: LogInViewProtocol {
         errorLabel.isHidden = false
```

Comment:
configuring errorLabel is logic and should be called from presenter via protocol